src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/actionplan/GoalResourceMapper.kt

```diff
@@ -66,7 +69,7 @@ class GoalResourceMapper(
       updatedAt = instantMapper.toOffsetDateTime(lastUpdatedAt)!!,
       updatedAtPrison = lastUpdatedAtPrison,
       notes = notes,
-      goalNotes = emptyList(),
```

Comment:
This was essentially how it worked before - the goalResourceMapper would return a GoalResource where the goalNotes field was always an emptyList
And then the client code was responsible for mapping the notes into NotesResource's, and then bodging them into the GoalResource via a copy operation.
Nasty! 🤣